### DISCUSSION TITLE
Relating Concentration to Capacity

### DISCUSSION THREAD

**Discussion by mlnsai** [pybamm-team/PyBaMM#2151] [Q&A]
Hi Team,

For a 5 Ah cell (LGM50 - Chen2020 parameters),

The initial total lithium concentration turns out to be 0.289333815092399 mol which is around 7.754 Ah 
(Ah = mol * F *(1/3600))

1. How can I relate it to arrive at 5Ah?
2.  Also, the Total lithium concentration = Total Li in particles + Total Li in electrolyte => Does particles mean positive and negative electrode concentration combined?

However, I can relate the cell capacity of 5 Ah from capacities of negative and positive electrodes through their SOC ranges.

1. SOC range of negative electrode = 91.05 - 3.46 % = 87.59%
    Cell capacity = (SOC range) x Negative electrode capacity = 87.59 x 5.83 = 5.10 Ah

2. SOC range of positive electrode = 84.84% - 26.38% = 58.45%
    Cell capacity = (SOC range) x Positive electrode capacity = 58.45 x 8.741= 5.10 Ah

I hope I am correct. 

Thanks in advance :)

**Comment by brosaplanella** [comment]
1. Your calculations below on the SOC range seem correct (not sure where the 7.754 Ah comes from). Baiscally, the 5 Ah is the nominal capacity (at a given C-rate) so the theoretical capacity you calculate below should be a bit higher (as it is).
2. Yes, the total Li in particles refers to both electrodes.

**Reply by mlnsai** [reply]
Thanks Ferran. That clears a lot. 

However, the "Total Lithium [mol]" can be obtained from experiments. I converted that to capacity [A.h] using the formula i mentioned.

[A.h] = [mol] * self.param.F / 3600,

where, F is Faraday's number: 96485.3321 coulombs.

This is where i couldn't understand, how the cell capacity is around 5 Ah when the total Li is around 7.75 Ah (converted 0.28933 mol to Ah). Any lead would help me a lot.

Please have a look :)

**Reply by valentinsulzer** [reply]
Section 3.2.1 of [this paper](https://www.sciencedirect.com/science/article/pii/S0378775319303593?casa_token=uu19wir6iI4AAAAA:AzJwY_6PMT3iEBAuZjFPUK0GdhYDlogL2hgsVIz9AgUDUgUVAoGEPJ8TG3ayKY-SAKsaCElL) is a really nice explanation of how theoretical capacity can be calculated from electrode capacities, total lithium, and voltage limits. 

Alternatively see Appendix B of [this paper](https://iopscience.iop.org/article/10.1149/1945-7111/ac3e48/pdf)

**Reply by mlnsai** [reply]
That's great @tinosulzer . I will check these articles out. Thanks for the reply :) @brosaplanella
